crates/astria-sequencer/src/fees/storage/keys.rs

```diff
@@ -51,8 +43,26 @@ fn extract_asset_from_key(key: &str, prefix: &str) -> eyre::Result<IbcPrefixed>
 
 #[cfg(test)]
 mod tests {
-    use astria_core::primitive::v1::asset::Denom;
+    use astria_core::{
+        primitive::v1::asset::Denom,
+        protocol::transaction::v1::action::{
+            BridgeLock,
+            BridgeSudoChange,
+            BridgeUnlock,
+            FeeAssetChange,
+            FeeChange,
+            IbcRelayerChange,
+            IbcSudoChange,
+            Ics20Withdrawal,
+            InitBridgeAccount,
+            RollupDataSubmission,
+            SudoAddressChange,
+            Transfer,
+            ValidatorUpdate,
+        },
```

Comment:
nit:
```suggestion
        protocol::transaction::v1::action::*,
```

Reply:
I generally prefer to avoid wildcard imports outside of `use super::*;` in most `tests` modules as they can bring more into scope than intended (with traits potentially being the most troublesome).

Reply:
No worries! Just commented because I saw it had been done in some of the other tests which used all actions